src/main/java/io/github/thebusybiscuit/slimefun4/implementation/tasks/AncientAltarTask.java

```diff
@@ -140,10 +143,14 @@ private void abort() {
 
     private void finish() {
         if (running) {
-            dropLocation.getWorld().playSound(dropLocation, Sound.ENTITY_ZOMBIE_VILLAGER_CURE, 1F, 1F);
-            dropLocation.getWorld().playEffect(dropLocation, Effect.STEP_SOUND, Material.EMERALD_BLOCK);
-            dropLocation.getWorld().dropItemNaturally(dropLocation.add(0, -0.5, 0), output);
 
+            AncientAltarOutputEvent ancientAltarOutputEvent = new AncientAltarOutputEvent(output, altar);
+            Bukkit.getPluginManager().callEvent(ancientAltarOutputEvent);
```

Comment:
Hmm. Maybe we have a pre and post event or something. I don't really like this running entirely and then not doing anything. 

Hmm.

Reply:
Huh? What do you mean?